datadog-sqs/datadog_monitor.tf

```diff
@@ -1,6 +1,9 @@
 locals {
   count           = var.env == "production" ? 1 : 0
   identifierLabel = var.identifier != "" ? "(${var.identifier})" : ""
+
+  oncall_handle        = "@webhook-rootly"
+  slack_warning_handle = "@slack-platform-warnings"
```

Comment:
Tady ty sqs monitory chodily do `#alerts` kde je i NY. Imho to dává smysl spíš v `#platform-warnings` kde není. 😄 (Já měl s těma kanálama nějakej záměr a nenapsal jsem si to, tak už nevím jak to bylo myšleno. 😅 Ale nejspíš tak, že v `#alerts` by zůstaly jen ty alerty který můžou v NY něco opravdu říct. Ale nevím jistě co by tak mělo být. Možná ta latence co tam dneska naskákala? Nebo co myslíš? @Crhaj

Reply:
Jo, tak to jsme alignuti. 😄 👍